### PR TITLE
Initial github issues template file

### LINKED ISSUE #29: Add issue template file
Add issue templates that list the steps for submitting an issue.

### PR DESCRIPTION
**Pull request by eldebrim**
Will resolve #29. This can be verified by submitting issues to this [taster fork](https://github.com/eldebrim/openstack_taster). Is there anything else that we should expect from issue submissions?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+If possible, please include the following information when submitting an issue:
+
+1. All Openstack environment variables
```
Comment (knightsamar) [inline .github/ISSUE_TEMPLATE.md]: "OpenStack"

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+2. The output of using `gem list`
+3. The log files in your openstack_taster/logs directory
```
Comment (knightsamar) [inline .github/ISSUE_TEMPLATE.md]: So openstack_taster will actually just create a `logs` directory. The `openstack_taster` parent dir is not required for running openstack_taster once it is installed as a gem.

**Review by knightsamar — COMMENTED** [review]
Thanks for making this.

Two more things we need to put here:

1. Recommend using `pastebin.osuosl.org` or some other pastebin service.

2. We should ask for their OpenStack version too.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (knightsamar) [reply .github/ISSUE_TEMPLATE.md]: On second thoughts, we won't need these. Ideally, they should declare all the variables from README and `openstack_taster` should work when pointed to any OpenStack deployment.

**Review by knightsamar — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+If possible, please include the following information in a pastebin such as `pastebin.osuosl.org` when submitting an issue:
```
Comment (knightsamar) [inline .github/ISSUE_TEMPLATE.md:1]: The pastebin is good for logs and `gem list` only. Everything else can remain in the issue text.

Actually, I think they can also attach the log files to an issue. That or pastebin, anything is preferable over pasting things in the issue.

**Review by knightsamar — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
...
-2. The output of using `gem list`
-3. The log files in your logs/ directory
-4. The steps required to reproduce the problem
+* The output of using `gem list`
+* The log files in your logs/ directory, you can attach the logs to the issue
+
+Include the following information in the issue submission:
+
+* Your OpenStack version
+* The steps required to reprodocue the problem
```
Comment (knightsamar) [inline .github/ISSUE_TEMPLATE.md]: *"reproduce" :)

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -1,6 +1,10 @@
 If possible, please include the following information in a pastebin such as `pastebin.osuosl.org` when submitting an issue:
 
-1. Your OpenStack version
-2. The output of using `gem list`
-3. The log files in your logs/ directory
-4. The steps required to reproduce the problem
+* The output of using `gem list`
```
Comment (knightsamar) [inline .github/ISSUE_TEMPLATE.md]: `using` can be dropped here.

**Review by knightsamar — CHANGES_REQUESTED** [review]
Just some final changes and we should be good!

**Review by hodgiwabi — APPROVED** [review]

**Comment by Codysseus** [comment]
Should we write something about ensuring their environment variables are correct?

**Comment by knightsamar** [comment]
@DesertBeagle I think it's safe to assume that they do have them correct, given that's the first instruction in the README. If we see a pattern later, we can do that.

**Review by Codysseus — APPROVED** [review]

**Review by knightsamar — APPROVED** [review]

**Pull request merged**